src/main/webapp/assets/content/json/irc.json

```diff
@@ -351,6 +351,78 @@
       "linkedin": "charininanayakkara"
     }
   },
+  {
+    "name": "Sanjanee Waniganeththi",
+    "description": "Nuclear Physics",
+    "image": "sanajee.jpg",
+    "social": {
+      "linkedin": "sanjanee-waniganeththi-92b3a3111"
+    }
+  },
+  {
+    "name": "Scynthya Nirukshan",
+    "description": "Bioscience Engineering",
+    "image": "scynthya.jpg",
+    "social": {
+      "linkedin": "gertrude-scynthya-nirukshan-a213476a"
+    }
+  },
+  {
+    "name": "Sumudu Herath",
+    "description": "Civil Engineering",
+    "image": "sumudu.jpg",
+    "social": {
+      "linkedin": "sumuduherath"
+    }
+  },
+  {
+    "name": "Maneesha Perera",
+    "description": "Engineering",
+    "image": "maneesha.jpg",
+    "social": {
+      "linkedin": "maneeshaperera"
+    }
+  },
+  {
+    "name": "Madhushi Bandara",
+    "description": "Computer Science",
+    "image": "madhusi.jpg",
```

Comment:
One of the previous images is overwritten by this image. Could you please rename this image to madhushibandara.jpg and restore the deleted image(https://sefglobal.org/assets/img/profiles/madhushi.jpg)